internals/scripts/clean.js

```diff
@@ -27,27 +27,35 @@ rm('-rf', 'app/components/*');
 // Cleanup containers folder
 rm('-rf', 'app/containers/*');
 mkdir('app/containers/App');
-mkdir('app/components/NotFoundPage');
-mkdir('app/components/HomePage');
+mkdir('app/containers/NotFoundPage');
+mkdir('app/containers/HomePage');
 cp('internals/templates/appContainer.js', 'app/containers/App/index.js');
-cp('internals/templates/notFoundPage.js', 'app/components/NotFoundPage/index.js');
-cp('internals/templates/homePage.js', 'app/components/HomePage/index.js');
+cp('internals/templates/notFoundPage.js', 'app/containers/NotFoundPage/index.js');
+cp('internals/templates/homePage.js', 'app/containers/HomePage/index.js');
 
 // Cleanup sagas folder
 rm('-rf', 'app/sagas/*');
 cp('internals/templates/sagas.js', 'app/sagas/index.js');
 
-// Cleanup selectors folder
+// Cleanup selectors folder, leave selectLocationSelection(.test).js
```

Comment:
To clarify, the comment is no longer necessary because we want clean to remove all selectors--which is problematic as selectLocationSelector is required for how we've setup react-router-redux, or just remove the extra info and keep the cp?